go-selinux/label/label_linux.go

```diff
@@ -47,7 +47,8 @@ func InitLabels(options []string) (plabel string, mlabel string, retErr error) {
 		}
 		for _, opt := range options {
 			if opt == "disable" {
-				return "", mountLabel, nil
+				selinux.ReleaseLabel(mountLabel)
+				return "", selinux.PrivContainerMountLabel(), nil
```

Comment:
This always returns `PrivContaienrMountLabel`, irregardless if the container is privileged or not?

Does this need an updated description of `InitLabels()` (and are consumers of this function required to make changes?); current description is:

```
// InitLabels returns the process label and file labels to be used within
// the container.  A list of options can be passed into this function to alter
// the labels.  The labels returned will include a random MCS String, that is
// guaranteed to be unique.
```

Reply:
If the selinux is disabled, then by definition it is Privileged from an SELinux point of view.  The goal is to always apply a label to the content, so that confined containers can not write to it.  Currently if we return "". The label might be read/writable by other containers.

Reply:
@thaJeztah Updated the docs.